rdsn/src/common/replication_common.h

```diff
@@ -117,6 +117,11 @@ class replication_options
     ~replication_options();
 
     void initialize();
+
+    int32_t min_2pc_replica_count(int32_t app_max_replica_count)
+    {
+        return std::min(mutation_2pc_min_replica_count, app_max_replica_count) - 1;
```

Comment:
Should be `return std::min(mutation_2pc_min_replica_count, app_max_replica_count - 1);` ?